### DISCUSSION TITLE
How to enable the New Architecture in RNW 0.68

### DISCUSSION THREAD

**Discussion by ndNovaDev** [microsoft/react-native-windows#9793] [Q&A] [answered]
the New Architecture is ready for use in RN 0.68 https://reactnative.dev/docs/new-architecture-intro  
Is there a way to enable it in RNW?

**Comment by chrisglein** [comment, marked as answer]
The short answer is that RNW 0.68 does not yet support the new architecture in full.
Turbo Modules are pretty far along, Fabric is a bit further out. But we owe documentation on this regardless.
